generators/ruby-v2/base/src/context/AbstractRubyGeneratorContext.ts

```diff
@@ -46,6 +46,23 @@ export abstract class AbstractRubyGeneratorContext<
         return this.customConfig.module ?? snakeCase(this.config.organization);
     }
 
+    public getVersionFromConfig(): string | undefined {
+        return this.config.output.mode._visit<string | undefined>({
+            publish: (gpc) => gpc.version || undefined,
+            downloadFiles: () => undefined,
+            github: (gom) => gom.version || undefined,
```

Comment:
nit: code is more readable with fuller variable names. `gom` -> `githubOutputMode`